rovercode/app.py

```diff
@@ -75,15 +87,16 @@ def on_message(_, raw_message):
                                    message[constants.MOTOR_VALUE_FIELD],
                                    message[constants.MOTOR_DIRECTION_FIELD])
     elif message_type == constants.CHAINABLE_RGB_LED_COMMAND:
-        try:
-            CHAINABLE_RGB_MANAGER.set_led_color(
-                message.get(constants.CHAINABLE_RGB_LED_ID_FIELD),
+        led_id = message.get(constants.CHAINABLE_RGB_LED_ID_FIELD)
+        LOGGER.info(f'Queueing RGB LED command to LED {led_id}')
+        GROVEPI_QUEUE.append(
+            functools.partial(
+                CHAINABLE_RGB_MANAGER.set_led_color, led_id,
                 message.get(constants.CHAINABLE_RGB_LED_RED_VALUE_FIELD),
                 message.get(constants.CHAINABLE_RGB_LED_GREEN_VALUE_FIELD),
-                message.get(constants.CHAINABLE_RGB_LED_BLUE_VALUE_FIELD))
-        except ValueError as exception:
```

Comment:
Since this was being handled before, you should probably move it up to the `action()` call to operate the same way